### PR TITLE
feat: filter for task description and dslText

### PR DESCRIPTION
**Pull request by klopfdreh**
Frontend implementation for: https://github.com/spring-cloud/spring-cloud-dataflow/issues/5294#issuecomment-1549828455

Needs to be integrated with: https://github.com/spring-cloud/spring-cloud-dataflow/pull/5344

Unfiltered:
Bildschirmfoto 2023-05-16 um 19 50 02

Filtered for description:
Bildschirmfoto 2023-05-16 um 19 50 13

Filtered for description and taskName:
Bildschirmfoto 2023-05-16 um 19 53 46

### CODE REVIEW

**Comment by klopfdreh** [comment]
I forgot to limit the selector only for dark theme. I am going to do so, soon.

**Comment by klopfdreh** [comment]
@claudiahub - is there anything to be changed? The backend implementation is already integrated. If the frontend part is also ok this feature could be used in the new SCDF version. 😃

**Comment by claudiahub** [comment]
@klopfdreh The frontend part looks good to me, I was wondering why you used `description` in the test for the parameter name and `descriptions` in the component (e.g. task-component.ts). 
Other than that this is good to be merged for the new scdf release. thank you

File: ui/src/app/tasks-jobs/tasks/tasks.component.ts [hunk 1]
```diff
@@ -47,6 +47,8 @@ export class TasksComponent extends DatagridComponent {
           params.current - 1,
           params.size,
           params?.taskName || '',
+          params?.descriptions || '',
```
[the remark this answers is not in the document]
Comment (klopfdreh) [reply ui/src/app/tasks-jobs/tasks/tasks.component.ts:50]: Yes - I named it wrong 👍 - feel free to just rename it after the merge. Thanks a lot for the review!

[the remark this answers is not in the document]
Comment (klopfdreh) [reply ui/src/app/tasks-jobs/tasks/tasks.component.ts]: To save you some time I did this already. 😄

**Pull request merged**